community/server/src/docs/ops/powershell.asciidoc

```diff
@@ -4,7 +4,7 @@
 The Neo4j PowerShell module allows administrators to:
 
 * install, start and stop Neo4j Windows® Services
-* and start tools, such as `Neo4j Shell` and `Neo4j Import`.
+* and start tools, like `Neo4j Import`.

```

Comment:
Please revert "like" to "such as" (http://dictionary.cambridge.org/grammar/british-grammar/such-as)